packages/components/src/toggle-group-control/test/index.tsx

```diff
@@ -143,19 +140,29 @@ describe.each( [
 			'Click for Delicious Gnocchi'
 		);
 
-		await user.hover( firstRadio );
+		await hover( firstRadio );
 
-		const tooltip = await screen.findByText(
-			'Click for Delicious Gnocchi'
-		);
+		const tooltip = await screen.findByRole( 'tooltip', {
+			name: 'Click for Delicious Gnocchi',
+		} );
 
 		await waitFor( () => expect( tooltip ).toBeVisible() );
 
-		await cleanupTooltip( user );
+		// hover outside of radio
+		await hover( document.body );
+		await hover( document.body, { clientX: 10, clientY: 10 } );
```

Comment:
Why do we need two `hover` calls?

Reply:
That is something that I derived from looking at how ariakit tests are written, and it is used to simulate hovering outside an element. I refactored it to a separate utility function to better clarify. [`ec59f15` (#57547)](https://github.com/WordPress/gutenberg/pull/57547/commits/ec59f150e0c9ea0143b43c37f542de047b87d59b)